dockerfiles/install_common_deps.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
is this file supposed to be part of the checkin? isn't this the same as the one in dockerfiles/scripts ?

Reply:
yes, you are correct.